### PR TITLE
travis.yml: update xcode to 9.4

### PR DESCRIPTION
**Pull request by commitay**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [ ] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
Some weird Travis RVM stuff here.

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,22 @@
-language: c
+language: ruby
```
Comment (MikeMcQuaid) [inline .travis.yml]: @reitermarkus Would rather avoid this if possible; the boot time is slower. Perhaps `rvm: system` is sufficient?

Comment (commitay) [reply .travis.yml]: `language: ruby` seems to be required to be able to specify `rvm: system`

**Comment by MikeMcQuaid** [comment]
Ok, if that's the only option begrudging (to Travis not you) 👍. Can you update the homebrew-test-bot repo similarly? Thanks!

**Comment by MikeMcQuaid** [comment]
@commitay Looks like this broke code coverage uploading. Mind either taking a look or reverting? Thanks 🙇

**Comment by commitay** [comment]
https://github.com/Homebrew/homebrew-test-bot/pull/167

**Pull request merged**